apiml-security-common/src/main/java/org/zowe/apiml/security/common/handler/ServletErrorUtils.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package org.zowe.apiml.security.common.handler;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.experimental.UtilityClass;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.zowe.apiml.message.api.ApiMessageView;
+import org.zowe.apiml.message.core.MessageType;
+import org.zowe.apiml.message.log.ApimlLogger;
+
+import java.io.IOException;
+import java.util.function.BiConsumer;
+
+@UtilityClass
+public class ServletErrorUtils {
+
+    public static BiConsumer<ApiMessageView, HttpStatus> createApiErrorWriter(HttpServletResponse response, ApimlLogger logger) {
+        return (apiMessageView, status) -> {
+            response.setStatus(status.value());
+            if (apiMessageView != null) {
```

Comment:
Is it an intentional response without a body? I guess we have message ZWEAM102 for that (see `org.zowe.apiml.message.core.AbstractMessageService#getInvalidMessageTemplate`). At least we should log what message is missing.

Reply:
That's true, the message object should always return a non-null value. I assume that if any code calls this lambda with `null` it is a wrong implementation or the test. Is there a use case for the null?

Reply:
probably not, just additional safety check